.travis.yml

```diff
@@ -22,7 +23,7 @@ env:
   - DISPLAY=:99 PATH=/usr/lib/chromium-browser:$PATH
 
 install:
-  - pip install -v .[prompt,vis]
+  - pip install .[prompt,vis]
```

Comment:
That's going to conflict with https://github.com/Microsoft/TextWorld/pull/13 which already fixes it

Reply:
Ok reverting that change.

Reply:
What about adding Python 3.7 ?

Reply:
haha got my answer from Travis:
"Unable to download 3.7 archive. The archive may not exist. Please consider a different version."

Reply:
Haha yep, gotta wait for Travis to support it.  Well, we could manually set up a Python environment on Travis but let's not